src/components/cc-orga-member-list/cc-orga-member-list.smart.md

```diff
@@ -59,4 +61,7 @@ interface ApiConfig {
 }'>
     <cc-orga-member-list></cc-orga-member-list>
 <cc-smart-container>
-```
\ No newline at end of file
+```
+<cc-smart-container context='{ "ownerId": "orga_3547a882-d464-4c34-8168-add4b3e0c135" }'>
+  <cc-orga-member-list></cc-orga-member-list>
+</cc-smart-container>
```

Comment:
TODO @florian-sanders-cc, this is only here for testing, remove once the review is done.